### DISCUSSION TITLE
adblock is not working?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7076] [Q&A]
ı watch youtube video and search something but adblock system not wotking why??? cool browser but ı hate ad

**Comment by mschilli87** [comment]
Could you please share the output of `:version` and `:config-diff`?
Also have a look at [known adblock issues](https://github.com/qutebrowser/qutebrowser/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+adblock), in particular https://github.com/qutebrowser/qutebrowser/issues/5754 and https://github.com/qutebrowser/qutebrowser/issues/6480 .

**Comment by ghost** [comment]
Qt: 5.15.2

CPython: 3.9.2
PyQt: 5.15.2

sip: 5.4.0
colorama: no
jinja2: 2.11.3
pygments: no
yaml: 5.3.1
adblock: no
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.2
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.34.1
QtNetwork SSL: OpenSSL 1.1.1k  25 Mar 2021

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Intel Open Source Technology Center, 3.0 Mesa 20.3.5
Platform: Linux-5.10.0-12-amd64-x86_64-with-glibc2.31, 64bit
Linux distribution: Debian GNU/Linux 11 (bullseye) (debian)




colors.completion.category.bg = #240046
colors.completion.category.border.bottom = #3c096c
colors.completion.category.border.top = #3c096c
colors.completion.category.fg = #c77dff
colors.completion.even.bg = #5a189a
colors.completion.fg = #dab6fc
colors.completion.item.selected.bg = #3c096c
colors.completion.item.selected.border.bottom = #757bc8
colors.completion.item.selected.border.top = #757bc8
colors.completion.item.selected.fg = #9d4edd
colors.completion.match.fg = #757bc8
colors.completion.odd.bg = #5a189a
colors.completion.scrollbar.bg = #240046
colors.completion.scrollbar.fg = #c77dff
colors.downloads.bar.bg = #5a189a
colors.downloads.error.bg = #240046
colors.downloads.error.fg = #c77dff
colors.downloads.stop.bg = #5a189a
colors.downloads.system.bg = none
colors.hints.bg = #5a189a
colors.hints.fg = #c77dff
colors.hints.match.fg = #c77dff
colors.keyhint.bg = #240046
colors.keyhint.fg = #9d4edd
colors.keyhint.suffix.fg = #c77dff
colors.messages.error.bg = #3c096c
colors.messages.error.border = #757bc8
colors.messages.error.fg = #c77dff
colors.messages.info.bg = #3c096c
colors.messages.info.border = #757bc8
colors.messages.info.fg = #c77dff
colors.messages.warning.bg = #3c096c
colors.messages.warning.border = #757bc8
colors.messages.warning.fg = #c77dff
colors.prompts.bg = #5a189a
colors.prompts.border = 1px solid #757bc8
colors.prompts.fg = #c77dff
colors.prompts.selected.bg = #3c096c
colors.statusbar.caret.bg = #5a189a
colors.statusbar.caret.fg = #c77dff
colors.statusbar.caret.selection.bg = #5a189a
colors.statusbar.caret.selection.fg = #c77dff
colors.statusbar.command.bg = #240046
colors.statusbar.command.fg = #c77dff
colors.statusbar.command.private.bg = #240046
colors.statusbar.command.private.fg = #c77dff
colors.statusbar.insert.bg = #5a189a
colors.statusbar.insert.fg = #c77dff
colors.statusbar.normal.bg = #3c096c
colors.statusbar.normal.fg = #c77dff
colors.statusbar.passthrough.bg = #240046
colors.statusbar.passthrough.fg = #c77dff
colors.statusbar.private.bg = #240046
colors.statusbar.private.fg = #c77dff
colors.statusbar.progress.bg = #240046
colors.statusbar.url.error.fg = #c77dff
colors.statusbar.url.fg = #c77dff
colors.statusbar.url.hover.fg = #c77dff
colors.statusbar.url.success.http.fg = #c77dff
colors.statusbar.url.success.https.fg = #9d4edd
colors.statusbar.url.warn.fg = #c77dff
colors.tabs.bar.bg = #3c096c
colors.tabs.even.bg = #240046
colors.tabs.even.fg = #757bc8
colors.tabs.indicator.error = #5a189a
colors.tabs.indicator.system = none
colors.tabs.odd.bg = #240046
colors.tabs.odd.fg = #757bc8
colors.tabs.selected.even.bg = #5a189a
colors.tabs.selected.even.fg = #c77dff
colors.tabs.selected.odd.bg = #5a189a
colors.tabs.selected.odd.fg = #c77dff
https://webcamtoy.com: content.media.video_capture = true
https://9convert.com: content.notifications = false
https://www.reddit.com: content.notifications = false
https://www.youtube.com: content.notifications = false
https://mail.google.com?extsrc=mailto&url=%25s: content.register_protocol_handler = false

**Reply by mschilli87** [reply]
@ArchKubi:
> ```
> adblock: no
> ```

It seems like you installed qutebrowser without adblock support.
~I don't use Arch myself and don't feel like reading their documentation for you but~ I am sure there is a way to get [python-adblock](https://github.com/ArniDagur/python-adblock) installed and that might be all that's missing.

**Reply by The-Compiler** [reply]
Despite @ArchKubi's name, "Linux distribution: Debian GNU/Linux 11 (bullseye) (debian)" seems like a rather weird Arch derivative :wink:

**Reply by ghost** [reply]
ı'am not use arch now ı test debian 
is this joke my english not good 
ı install python-adblock but youtube ad not block

**Reply by mschilli87** [reply]
Does `:version` say `adblock: yes` now?

**Reply by The-Compiler** [reply]
Note that youtube ads won't be blocked either way, see #6480 already linked by @mschilli87 above.
